.github/workflows/on_docs_update.yml

```diff
@@ -1,22 +1,23 @@
 name: "On docs update"
 on:
   push:
+    branches:
+    - develop
     paths:
     - 'docs/**'
```

Comment:
I didn't know if this would behave as an `OR` or as an `AND`, that's why I restricted it from the step level, but since the job does not appear here it means it works as an `AND`! Thanks!